packages/lib/callbackUrlUtils.test.ts

```diff
@@ -30,6 +30,12 @@ describe('callbackUrlUtils', function() {
 		expect(tagUrl).toBe('joplin://x-callback-url/openTag?id=123456');
 	});
 
+	it('should build valid install plugin callback urls', () => {
+		const installPluginUrl = callbackUrlUtils.getTagCallbackUrl('test.plugin-joplin');
+		expect(callbackUrlUtils.isCallbackUrl(installPluginUrl)).toBe(true);
+		expect(installPluginUrl).toBe('joplin://x-callback-url/openTag?id=test.plugin-joplin');
+	});
```

Comment:
why are you testing tag urls here

Reply:
Sorry, I borrowed the code from the above test, and forgot to change the name ; )